crates/holochain/src/core/ribosome/host_fn/agent_info.rs

```diff
@@ -8,16 +8,16 @@ use std::sync::Arc;
 pub fn agent_info<'a>(
     _ribosome: Arc<impl RibosomeT>,
     call_context: Arc<CallContext>,
-    _input: AgentInfoInput,
-) -> RibosomeResult<AgentInfoOutput> {
+    _input: (),
```

Comment:
Same note as below: maybe use an empty enum here that can be extended in the future? I'm suggesting this because we can use https://blog.rust-lang.org/2019/12/19/Rust-1.40.0.html#non_exhaustive-structs-enums-and-variants to make it future-proof

Reply:
the agent info is supposed to be deterministic based on the current source chain so there shouldn't be any input to it